src/OpenTelemetry.Exporter.Prometheus.Shared/README.md

```diff
@@ -11,8 +11,18 @@
 
 ### Step 1: Install Package
 
+Depending on the hosting mechanism you will be using:
```

Comment:
why would shared project has a readme? we dont release it anyway...

Reply:
removed